### ISSUE TITLE
Stuck in reconnection screen

### ISSUE THREAD

**Issue by moufmouf** [thecodingmachine/workadventure#181]
Seen by @npeguin 

While he was away with WA an another tab, the Reconnection Screen got displayed and WA is stuck in reconnection.

Logs:

```
Switching to disconnected scene
Reconnecting to game scene from setTimeout
Reconnect called but setTimeout in progress for the reconnection
```

**Comment by moufmouf** [comment]
Note: we don't listen to the reconnect events of socket.io and in particular, not to the reconnect_error event.

https://socket.io/docs/client-api/#Event-%E2%80%98reconnect%E2%80%99-1

**Comment by moufmouf** [comment]
See #184
Closed by #185
